libs/components/forms/src/lib/modules/radio/radio.component.scss

```diff
@@ -5,6 +5,14 @@
   display: inline-block;
 }
 
+.sky-radio-help-inline {
+  display: inline-flex;
+}
+
+.sky-switch-control sky-margin-inline-xs {
```

Comment:
Should `sky-margin-inline-xs` have a period? `.sky-margin-inline-xs`

Reply:
ah man, i thought that had fixed a spacing issue i had seen, but i don't think this solution i have now actually fixed it. pushing up a different idea up now